postgres/matviews/replication_docs.sql

```diff
@@ -0,0 +1,16 @@
+CREATE MATERIALIZED VIEW public.app_monitoring_replication_docs
+TABLESPACE pg_default
+AS
+  SELECT
+    created,
+    partner_name,
+    COALESCE((doc #>> '{couchdb,medic,doc_count}')::int, 0) AS doc_count
```

Comment:
seems weird to make a matview for just one metric.. but it seems to be what we're doing elsewhere in this codebase. 

any thoughts on this? should we merging view with others like it (eg. `users_replication_limit.sql` or `sms_error_states.sql`)?  cc @eljhkrr @freddieptf

Reply:
@kennsippell  Yes, you are absolutely right, we can group them in one matview.

Reply:
I can make that change in this PR. The query takes 866 ms (3ms fetch)